www/routes/raw.ts

```diff
@@ -37,23 +48,9 @@ export const handler: Handlers = {
       return response;
     }
 
-    if (isHTML) {
-      response.headers.set("Content-Type", "text/plain");
-    } else if (path.endsWith(".ts")) {
-      response.headers.set("Content-Type", "application/typescript");
-    } else if (path.endsWith(".js")) {
-      response.headers.set("Content-Type", "application/javascript");
-    } else if (path.endsWith(".tsx")) {
-      response.headers.set("Content-Type", "text/tsx");
-    } else if (path.endsWith(".jsx")) {
-      response.headers.set("Content-Type", "text/jsx");
-    } else if (path.endsWith(".json")) {
-      response.headers.set("Content-Type", "application/json");
-    } else if (path.endsWith(".wasm")) {
-      response.headers.set("Content-Type", "application/wasm");
-    } else {
-      response.headers.set("Content-Type", "text/plain");
-    }
+      const value = contentTypes.get(extname(path)) ?? "text/plain";
```

Comment:
Very close! Now if `isHTML` is `true` we can bypass the `extname(path)` and directly assume a value of `.html`.

Reply:
Do you mean something like this? 
let value = isHTML ? "text/html" : contentTypes.get(extname(path));

Reply:
Close, if we read the existing code carefully, it returns `text/plain` for html files, not `text/html`.